server/public/styles.css

```diff
@@ -23,16 +26,39 @@ body {
   border-radius: 10px;
 }
 
-.form-control,
-.message {
-  font-family: Quicksand, sans-serif;
+.input-group-text {
+  background-image: linear-gradient(#fff, #fff);
+}
+
+.fa-magnifying-glass,
+.fa-location-dot,
+.fa-location-crosshairs,
+.fa-phone {
+  font-size: 1rem;
 }
 
-.fas {
+.fa-magnifying-glass,
+.fa-location-dot,
+.fa-phone {
+  color: #fe4900;
+}
+
+.fa-location-dot {
+  width: 1rem;
+}
+
+.fa-location-crosshairs {
```

Comment:
Rather than augmenting library classes, define your own classes to use alongside them. Otherwise, the default behavior (the documentation) is no longer a reliable resource for teammates.